crates/subspace-farmer/src/single_disk_farm/plotting.rs

```diff
@@ -272,13 +294,15 @@ where
         modifying_sector_index.write().await.take();
 
         if replotting {
-            info!(%sector_index, "Sector replotted successfully");
+            debug!(%sector_index, "Sector replotted successfully");
```

Comment:
Can we have this as an `info`? I use these lines a lot in my work to determine that my local build works and IMHO it's good to see constant progress as part of UX.

Reply:
There are still info messages, just in different place, you can find them above